src/modals/WithdrawModal.js

```diff
@@ -264,6 +266,9 @@ class WithdrawModal extends React.Component {
           <I s="WithdrawInstructionNotificationFailed" />,
           this.props.theme
         );
+        if (this.state.notifyEnabled) {
+          window.open(`https://9000.hal.xyz/recipes/loopring-withdrawal-notification?useraddress=${this.props.dexAccount.account.address}`);
```

Comment:
Doesn't HAL offer an API that could help you avoid to redirect users to another page, and instead, register the notification directly on the Loopring UI?